<launch>
   <!-- args --> 
   <arg name="host_ip" default="$(env host_ip)" />
   <arg name="img_width" value="512"/>
    <arg name="img_height" value="288"/>
    <param name="ip_addr" type="string" value="$(arg host_ip)" />

1;2P

    <param name="stats_file_addr" value="$(find package_delivery)/../data/package_delivery/stats.json" />
    <param name="supervisor_mailbox" value="$(find package_delivery)/../data/package_delivery/supervisor_mailbox.txt" />

    <!-- arg name="localization_method" value="ground_truth" /-->
    <arg name="localization_method" value="gps" />
    <param name="localization_method" value="$(arg localization_method)" />

    <!-- setting DEBUG to false prevents publishing and subscribing to rviz /--> 
    <!-- also planning to prevent stdio stuff(if they really matter)--> 
    <arg name="DEBUG" value="true" /> 
    <param name="ros_DEBUG" type="bool" value="$(arg DEBUG)" />

    <node pkg="profile_manager" type="profile_manager_node" name="profile_manager" output="screen">
    <param name="stats_file_addr" value="$(find package_delivery)/../data/package_delivery/stats.json" />
    <param name="ip_addr" value="$(arg host_ip)"/>
    </node>

    <node pkg="control_drone" type="control_thread" name="control_thread" output="screen"/>
    <node pkg="control_drone" type="probe_thread" name="probe_thread" output="screen"/>

</launch>
